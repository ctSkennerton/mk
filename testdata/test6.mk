# Variable deps is set in include.
# in P9P, . is the cwd of where we execute mk
<./testdata/deps6

test3.mk.o: $deps
	secondprocess ab

one:
	process bar

two:
	process rebar
