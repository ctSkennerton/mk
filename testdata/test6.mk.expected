two: process rebar
one: process bar
test3.mk.o: secondprocess ab
